### PR TITLE
feat(page-filters): Add condensed option to page filter bar

### PR DESCRIPTION
**Pull request by davidenwang**
Add `condensed` option to page filter bar instead of rewriting `max-width: 100%; width: max-content`

### CODE REVIEW

File: static/app/views/replays/filters.tsx [hunk 1]
```diff
@@ -9,17 +9,15 @@ import space from 'sentry/styles/space';
 
 function ReplaysFilters() {
   return (
-    <StyledPageFilterBar>
+    <StyledPageFilterBar condensed>
       <ProjectPageFilter />
       <EnvironmentPageFilter />
-      <DatePageFilter />
+      <DatePageFilter alignDropdown="left" />
```
Comment (billyvg) [inline static/app/views/replays/filters.tsx:15]: Isn't left default?

**Review by ryan953 — COMMENTED** [review]
I think also `static/app/views/dashboardsV2/detail.tsx` is setting `width` & `max-width` (and margin-bottom) and could make use of the new prop

File: static/app/views/replays/filters.tsx (hunk 1 above)
Comment (davidenwang) [reply static/app/views/replays/filters.tsx:15]: not for the date page filter where when absolute date is selected it will align to the right.

**Comment by davidenwang** [comment]
> I think also `static/app/views/dashboardsV2/detail.tsx` is setting `width` & `max-width` (and margin-bottom) and could make use of the new prop

I'll add that in to this PR as well

**Review by evanpurkhiser — APPROVED** [review]

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**